### ISSUE TITLE
War Territory & Friendly Fire

### ISSUE THREAD

**Issue by Midnight9746** [magnusulf/Factions#53]
I am unsure if there's an option, but it would be great if there was an option to allow players in a Faction to fight each other. Most of the players PVP and events kinda get hindered by them being unable to attack each other. (Preferably if they can fight in add out of territorys)

The other thing, which I don't know would be added or considered is territories set by the server. Essentially a place on the server will be owned by no faction. (I see warzone and safezone but they're not really what I'm looking for) and players can challenge the territory by some means of mob killing or something and then the territory would say their faction owns the land. I don't want them being able to do things such as build and destroy, it's ment to be a way to challenge the players to be the best faction.

All of the player factions are set to be unclaimable. Meaning that no other Faction can take over the land. Each faction is basically a cabin of sorts. (Mythology server) so their land shouldn't be taken over. The server areas should be able to be taken over (pre-define territories owned by the server), but they won't block non-faction members from using anything there like a normal territory would such as npcs, doors, chests, etc.

TLDR;

- Being able to allow friendly fire
- Allowing server areas to be claimed

To the server area thing, the idea is basically what Wynncraft has, except the banner post of the region won't be there, instead you would challenge it by being inside of it.

**Comment by magnusulf** [comment]
1. Friendly fire can be enabled through Faction flags.
2. The other idea is not going to happen because I have to prioritise what I spend time on. I am sorry about that.
